python/ql/src/semmle/python/dataflow/new/internal/DataFlowPublic.qll

```diff
@@ -135,9 +136,14 @@ class Node extends TNode {
   LocalSourceNode backtrack(TypeBackTracker t2, TypeBackTracker t) { t2 = t.step(result, this) }
 
   /**
-   * Gets a local source node from which data may flow to this node in zero or more local steps.
+   * Gets a local source node from which data may flow to this node in zero or more local data-flow steps.
    */
   LocalSourceNode getALocalSource() { result.flowsTo(this) }
+
+  /**
+   * Gets a local source node from which data may flow to this node in zero or more local taint-flow steps.
```

Comment:
Same as my other comment. To avoid confusion, the QLDoc should indicate that the taint tracking libraries must be imported for this to have any results.

Reply:
Done, also mentioned in the change note.